jdaviz/configs/specviz2d/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -318,20 +318,24 @@ def _trace_dataset_selected(self, msg=None):
         default_bg_width = int(np.ceil(width / 10))
         default_width = min(default_bg_width, distance_from_edge * 2)
 
+        # sign for one-sided and single trace-pixel depending on whether the brightest pixel is
+        # above or below the middle of the image
+        if default_bg_width * 2 >= distance_from_edge:
+            sign = 1 if (brightest_pixel < width / 2) else -1
+            default_bg_separation = sign * default_bg_width * 2
```

Comment:
these lines will be covered by #1739 (coverage was lost when skipping the remote test in #1736)